code/controllers/subsystems/ticker.dm

```diff
@@ -0,0 +1,537 @@
+SUBSYSTEM_DEF(ticker)
+	name = "Ticker"
+	init_order = SS_INIT_TICKER
+
+	priority = SS_PRIORITY_TICKER
+	flags = SS_KEEP_TIMING
+	runlevels = RUNLEVEL_LOBBY | RUNLEVEL_SETUP | RUNLEVEL_GAME
+
+	var/current_state = GAME_STATE_STARTUP	//State of current round used by process()
+	var/force_ending = FALSE				//Round was ended by admin intervention
+	var/bypass_checks = FALSE 				//Bypass mode init checks
+	var/setup_failed = FALSE 				//If the setup has failed at any point
+
+	var/start_immediately = FALSE			//If true, there is no lobby phase, the game starts immediately.
+	var/setup_done = FALSE					//All game setup done including mode post setup and
+
+	var/list/datum/mind/minds = list()		//The people in the game. Used for objective tracking.
+
+	var/Bible_icon_state					// icon_state the chaplain has chosen for his bible
```

Comment:
this stuff definitely doesn't belong in the ticker

Reply:
Deleted.